config.js

```diff
@@ -8,13 +8,13 @@ module.exports = {
   manifestDisplay: 'standalone',
   manifestIcon: 'src/assets/img/website-icon.png',
   pathPrefix: `/gatsby-starter-grayscale/`, // This path is subpath of your hosting https://domain/portfolio
-  heading: 'Anubhav Srivastava',
-  subHeading: 'Full time Web Developer. Part time Open source contributor  ',
+  heading: 'Roger Gracie Academy',
+  subHeading: 'Brazilian Jiu-Jitsu Academy, Darlington.  ',
 
   // social
   socialLinks: [
     {
-      icon: 'fa-github',
+      icon: 'fa fa-instagram',
       name: 'Github',
       url: 'https://github.com/anubhavsrivastava',
```

Comment:
Do they have insta already? If so let's get that in in another PR (or here, up to you!) 👍

Reply:
So I've actually started a new branch `social-links` and tried to get the icons to work, but came across some issues, so I never committed . 
The links themselves work, but the icons for instagram and facebook do not. I think it's something to do with the version of Font Awesome Icons, where Insta and FB are not available anymore.